### ISSUE TITLE
Tinybase fails to build with Browserify

### ISSUE THREAD

**Issue by shaneosullivan** [tinyplex/tinybase#19]
### Describe the bug

When adding an `import "tinybase"` or `require("tinybase")` in a file that is then passed to Browserify for buliding, it fails with the error

```
Error: Can't walk dependency graph: Cannot find module 'tinybase' 
```

Perhaps it is related to bundler requirements, but I build by app with many other packages and they all work just fine.

### Your Example Website or App

https://github.com/shaneosullivan/tinybaseRepro

### Steps to Reproduce the Bug or Issue

1: Go to https://github.com/shaneosullivan/tinybaseRepro and check out the project
2: Run `npm install`
3: Run `npm run repro`

### Expected behavior

It should successfully build the file `output.js` without error

### Platform

- OS: macOS
- Browser: Terminal
- Version: 1.3.3

**Comment by jamesgpearce** [comment]
nice issue; let me take a look

**Comment by shaneosullivan** [comment]
Obligatory macro from my Gitmeme extension (https://gitme.me) :-)

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2FKJ1f5iTl4Oo7u%2Fgiphy-downsized-medium.gif&token=thanks" data-gitmeme-token="thanks"><img src="https://media1.giphy.com/media/KJ1f5iTl4Oo7u/giphy-downsized-medium.gif" title="Created by gitme.me with /thanks"/></a>

**Comment by jamesgpearce** [comment]
OK, so I think there are two problems.

First is that Browserify doesn't seem to support the modern `exports` field (see https://github.com/browserify/browserify/issues/1995).

So I got around that by specifying the path directly - with `require("tinybase/lib/umd/tinybase");` - to get to the UMD version of the library.

Secondly, Browserify does not seem particularly happy with the `??=` nullish assignment operator, which is pretty recent. Until it knows about that, you'll need to transform ahead of time.

I did this with Babel by installing this:

```
npm install --save-dev babelify @babel/core @babel/preset-env
```

...and then adding `-g [ babelify --presets [ @babel/preset-env ] ]` to the command line. The spaces are significant.

See this PR: https://github.com/shaneosullivan/tinybaseRepro/pull/1

When run:

```
$ node output.js 
{ greeting: { hi: 'there!' } }
```
